.github/workflows/main.yml

```diff
@@ -42,19 +46,20 @@ jobs:
       # Indicates the location of the vcpkg as a Git submodule of the project repository.
       VCPKG_ROOT: ${{ github.workspace }}/external/LibAPR/vcpkg
       CIBW_ENVIRONMENT_WINDOWS: EXTRA_CMAKE_ARGS="-DCMAKE_TOOLCHAIN_FILE=D:\\a\\PyLibAPR\\PyLibAPR\\external\\LibAPR\\vcpkg\\scripts\\buildsystems\\vcpkg.cmake -DVCPKG_MANIFEST_DIR=D:\\a\\PyLibAPR\\PyLibAPR\\external\\LibAPR\\"
-      CIBW_BUILD: "cp36-* cp37-* cp38-* cp39-*"
```

Comment:
Why did we remove 3.6?

Reply:
I think we can add 3.6 back in - but we'd need to drop the version requirement for pyqtgraph, as it doesn't support python 3.6. Do you remember why you added it?

Reply:
I tested locally with `pip install --no-cache-dir .` on 3.6 without the requirement. It installs pyqtgraph 0.11.1 and everything seems to run fine (interactive apr, viewer, raycast, tests)

Reply:
added it back in (55d5468dc068c6c3a857153765832fa6afa54bca)